README.md

```diff
@@ -1 +1,182 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Tech-Event] <a name="about-project"></a>
+
+>
+Tech-Event is a fully responsive website. This project considred as fist module capstone project. This peoject represent an event that is about technology and its rapid growth in human lief.
+
+
+## 🛠 Built With <a name="built-with"></a>
+HTML
+CSS-(Bootsrap framework)
+Javascript
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Simple design
+- clear Content
+- responsive
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here you can see a live demo of my portfolio
+
+- [Live Demo Link]( https://nilab-khwaja.github.io/first-capstone/)
+
+
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+### Prerequisites
+
+- Basic html, css and medium Javascript know how.
+- Git installation
+- A text editor 
+- A terminal
+- A web browser to view output e.g Google Chrome
+- An IDE e.g Visual studio code
+
+### Setup
+
+- Clone this repository or download the Zip folder:
+
+```
+git clone https://github.com/nilab-khwaja/first-capstone.git
+```
+
+- Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd My_Portfolio``**
+
+### Usage
+
+- After Cloning this repo to your local machine
+- Open the `index.html` in your browser
+
+### Run Tests
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+npm install
+```
+Track HTML linter errors run:
+```
+npx hint .
+```
+Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+npx eslint .
+```
+
+### Deployment
+
+- I used GitHub Pages to deploy my website
+- For more information about publishing sources, see "[About github page](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#publishing-sources-for-github-pages-sites)"
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> 
+
+👤 **Author1**
+
+- GitHub: [@nilab-khwaja](https://github.com/githubhandle)
+- Twitter: [@nilab](https://twitter.com/twitterhandle)
+- LinkedIn: @nilab-khwajazada
+
+
+## Credits
+
+- The original design ideal [Microverse💕](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)
+- Project from [Microverse](https://bit.ly/MicroverseTN) html & css module
+
```

Comment:
- [ ] Great for giving credits but it would be great if you give the appropirate credit since the Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07). so please add it 🤩🔥✅

![image](https://user-images.githubusercontent.com/58553711/227087288-e5ba260b-bd6e-4b6f-a1d7-27690aa8fae8.png)